mockgcp/README.md

```diff
@@ -97,4 +99,18 @@ We will compare new HTTP logs with the golden HTTP log as a part of the mockGCP
 
 To prevent any problems when comparing with the golden logs, it is necessary to replace all generated values in the HTTP log with identical values.
 
-See [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/unified_test.go#L167-L329) to get some ideas.
\ No newline at end of file
+See [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/unified_test.go#L167-L329) to get some ideas.
+
+## Appendix
+
+### Generating proto
+
+If tbe proto file on [googleapis](https://github.com/googleapis/googleapis/commits/1e4137870560340a14700618a05e2d7162326af7/google/cloud/ids/v1/ids.proto) is out of date or non existent, you can generate a proto file from the [google generated api](https://github.com/googleapis/google-api-go-client/tree/b49e3b908a8ed562e068736f1c42e992538ba6e0) as such:
+
+```shell
+$ 	wget -O ids-api-v1.json https://raw.githubusercontent.com/googleapis/google-api-go-client/b49e3b908a8ed562e068736f1c42e992538ba6e0/ids/v1/ids-api.json
+	mkdir -p apis/mockgcp/cloud/ids/v1/
+	cd tools/gapic; go run . --proto-version=2 ../../ids-api-v1.json > ../../apis/mockgcp/cloud/ids/v1/service.proto
+```
+
+Modify for your own service and add it to the [`generate-proto-from-openapi`](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/bbdd7e244a8e9c1259ab939aa233c63fb38db1c2/mockgcp/Makefile#L73-L74).
```

Comment:
Keep the link to the master branch.
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/mockgcp/Makefile#L73-L74